pinc/theme.inc

```diff
@@ -124,7 +124,7 @@ function html_logobar()
 
     echo "\n<div id='header'>\n";
     echo "<div id='logo'>\n";
-    echo "<a href='$code_url'><img src='$code_url/graphics/dp-logo-360x68px.png' width='360' height='68' alt='$site_name'></a>\n";
+    echo "<a href='$code_url' />\n";
```

Comment:
Done this way the link does not work for me in Chrome. In my proposal I had the `<a>` wrap the `<div>`.